packages/subgraph/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "lib": ["DOM","ES2018"],
+    "types": ["@graphprotocol/graph-ts", "node"],
+    "skipLibCheck": true,                     /* Skip type checking of declaration files. */
+    "forceConsistentCasingInFileNames": true,  /* Disallow inconsistently-cased references to the same file. */
+    "strictNullChecks": true
+  },
+  "exclude": [
+    "src/types"
+  ]
```

Comment:
I think the last version inherited from the subgraph tsconfig, any reason to change that?

Reply:
I'll restore the inheritance:

```
{
  "compilerOptions": {
    "lib": ["DOM","ES2018"],
    "types": ["@graphprotocol/graph-ts", "node"],
    "typeRoots" : ["./node_modules/@types"],
    "skipLibCheck": true,                     /* Skip type checking of declaration files. */
    "forceConsistentCasingInFileNames": true,  /* Disallow inconsistently-cased references to the same file. */
    "strictNullChecks": true
  },
  "exclude": [
    "src/types",
    "node_modules"
  ],
  "extends": "@balancer-pool-metadata/tsconfig/base.json"
}

```